tsconfig.json

```diff
@@ -1,12 +1,6 @@
 {
 	"compilerOptions": {
-		"lib": [
-			"es2019",
-			"es2020.bigint",
-			"es2020.string",
-			"es2020.symbol.wellknown",
-			"es2020.promise"
-		],
+		"lib": ["es2020"],
```

Comment:
I think we should add the target es2019 to the lib. I added a link of an example from typescript repo on your comment in my last PR

Reply:
But doesn't es2020 include all of es2019 anyway?

Reply:
Yeah, It should. When I searched goggle, I found this repo https://github.com/tsconfig/bases/tree/main/bases and i noticed they always include the target version in the lib

Reply:
ok